src/ShoppinPal/Vend/Api/V2/Products.php

```diff
@@ -68,4 +68,33 @@ public function get($productId)
         return new Product($result['data'], Product::UNKNOWN_PROPERTY_IGNORE, true);
 
     }
+
+    /**
+     * Returns the true, that update image.
+     *
+     * @param string $productId ID of the product.
+     * @param string $imagePath of the image.
+     *
+     * @return true
+     *
+     * @throws EntityNotFoundException If the product is not found.
+     */
+    public function imageUpload($productId, $imagePath)
```

Comment:
You should expect a file stream resource or the content of the image here. The image may not be on the filesystem.